src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -466,6 +487,36 @@ public Builder set(String key, boolean value) {
         return set(key, ConfidenceValue.of(value));
       }
 
+      public Builder setIntegers(String key, java.util.List<java.lang.Integer> values) {
+        builder.put(key, ConfidenceValue.ofIntegers(values));
+        return this;
+      }
```

Comment:
Users can always create a ConfidenceValue.List object first, and pass that using:
```
Builder.set("key", list)
```
However these new accessory methods allow to create sub-lists in the structs by just using plain lists of Java types